docs/reference/developer/adr/0013-plot-should-not-resample-automatically

```diff
@@ -0,0 +1,67 @@
+ADR 0013: Plot should not resample automatically
+================================================
+
+- Status: accepted
+- Deciders: Jan-Lukas, Neil, Simon
+- Date: 2022-06-22
+
+Context
+-------
+
+The current plotting functionality automatically resamples 2-D (or higher) data as well as binned data of any dimension.
+There were three main drivers/motivations for this:
+
+- We frequently deal with data with "long" dimensions, such as "pixel" or "spectrum" dimensions, which may exceed a million.
+- We often deal with "ragged" data, i.e., data with 2-D coordinates that originate after coordinate transformations.
+- We frequently plot binned data.
+  Bins are often much larger than the target plot resolution, so histogramming must be done.
+  This is combined with "long" dimensions and potentially high resolutions along other dimensions (exceeding 1000 or even 10000).
+
+In the past we have therefore taken it as a given that resampling is a non-optional requirement.
+However, we have since then experienced a multitude of problems with this, ranging from simple bugs to complex and hard to maintain code.
+More importantly, the resulting plots are unpredictable and not faithful to the actual data.
+Concrete problems are:
+
+- Infinite zoom with event data is not useful and led to considerations on a different (more complicated) mechanism.
```

Comment:
Should we be more specific in describing in a few words what that mechanism was? (e.g. stopping the resampling at a given zoom level)